### DISCUSSION TITLE
Default Menu

### DISCUSSION THREAD

**Discussion by in4matix** [Schneegans/Fly-Pie#328] [Fly-Pie Q&A] [answered]
Thank you for this brilliant extension! How can I get back the default menu that comes with Fly-Pie? I accidentally deleted it while learning.

**Comment by Schneegans** [comment, marked as answer]
If you delete all of your menus and then logout / login, the default menu will be re-created. If you do not want to delete your menus, you can move them to the stash area instead and restore them from there after logging out / logging in.

**Reply by in4matix** [reply]
Thank you!
